sambacc/schema/conf-v0.schema.json

```diff
@@ -0,0 +1,303 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "mailto:phlogistonjohn+sambacc-v0@asynchrono.us",
```

Comment:
The value of the $id field has to be a unique URI according to the spec. Because SINK doesn't have a webpage nor did I want to point it back at a "raw" link on github I just used a mailto uri that just happens to be a valid contact for me. :-)